### PR TITLE
Adjust Spring Zeebe SDK docs

### PR DESCRIPTION
**Pull request by nicpuppa**
## Description

- Updated configuration properties
- Improved docs, removed reference to old configuration (with .properties file) in favor of yaml file
- Removed links to Spring Zeebe community project

closes: #3748 

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineerring DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

### CODE REVIEW

**Comment by nicpuppa** [comment]
> @nicpuppa It looks like #3748 is in review -- let me know when you would like a final review here, and if you have any particular deadline 👍

Thanks @christinaausley I will let you know when the PR is ready 👍

File: docs/apis-tools/spring-zeebe-sdk/configuration.md [hunk 1]
```diff
@@ -310,28 +437,54 @@ Read more about this feature in the [job streaming documentation](/apis-tools/ja
...
-zeebe.client.default-job-worker-stream-enabled=true
+```yaml
+camunda:
+  client:
+    zeebe:
+      defaults:
+        stream-enabled: true
+```
+
+This also works for every worker individual:
```
Comment (christinaausley) [inline docs/apis-tools/spring-zeebe-sdk/configuration.md:448]: Do you mean this also works for every job worker individually?

File: docs/guides/migrating-from-camunda-7/conceptual-differences.md [hunk 2]
```diff
@@ -149,7 +149,7 @@ Camunda 8 doesn't provide integration into Jakarta EE application servers like C
 
 ### CDI or OSGI
 
-Due to limited adoption, there is no support for CDI or OSGI in Camunda 8. A lightweight integration layer comparable to [Spring Zeebe](https://github.com/camunda-community-hub/spring-zeebe) might evolve in the feature, and we are happy to support this as a community extension to the Zeebe project.
+Due to limited adoption, there is no support for CDI or OSGI in Camunda 8. A lightweight integration layer comparable to the [Spring Zeebe SDK](../../apis-tools/spring-zeebe-sdk/getting-started.md) might evolve in the feature.
```
Comment (christinaausley) [inline docs/guides/migrating-from-camunda-7/conceptual-differences.md:152]: Do you mean a lightweight integration layer comparable to the Spring Zeebe SDK may be provided in the future?

File: docs/apis-tools/spring-zeebe-sdk/configuration.md (hunk 1 above)
Comment (ana-vinogradova-camunda) [reply docs/apis-tools/spring-zeebe-sdk/configuration.md:448]: Yes, this is my understanding.

File: docs/apis-tools/spring-zeebe-sdk/getting-started.md [hunk 3]
```diff
@@ -76,37 +76,63 @@ If you are using IntelliJ:
 Settings > Build, Execution, Deployment > Compiler > Java Compiler
 ```
 
-## Configuring the Zeebe cluster connection
+## Configuring Camunda 8 connection
```
Comment (koevskinikola) [inline docs/apis-tools/spring-zeebe-sdk/getting-started.md:79]: :upside_down_face: Just a suggestion

```suggestion
## Configuring the Camunda 8 connection
```

File: docs/apis-tools/spring-zeebe-sdk/configuration.md (hunk 1 above)
Comment (koevskinikola) [reply docs/apis-tools/spring-zeebe-sdk/configuration.md:448]: :upside_down_face: Should we fix the typo then?

```suggestion
This also works for every worker individually:
```

**Review by koevskinikola — APPROVED** [review]
I have some small suggestion, but the changes look good otherwise :+1:

**Review by christinaausley — APPROVED** [review]

**Review by koevskinikola — APPROVED** [review]

**Pull request merged**